src/brainbrowser/volume-viewer/volume-loaders/nifti1.js

```diff
@@ -392,22 +392,45 @@
     }
 
     var d = 0;                  // Generic loop counter.
-
-    // Convert data to bytes.
-    byte_data = new Uint8Array(native_data.length);
-    n_min = native_data[0];
-    n_max = native_data[0];
-
-    for (d = 1; d < native_data.length; d++) {
-      if (native_data[d] > n_max)
-        n_max = native_data[d];
-      if (native_data[d] < n_min)
-        n_min = native_data[d];
+    var value = 0;              // Scaled value.
+    var slope = header.scl_slope;
+    var inter = header.scl_inter;
+    var n_min, n_max;
```

Comment:
For line 398 to 429, what's about something like that: 

```javascript
var n_min =  Infinity; 
var n_max = -Infinity; 
 
if (slope !== 0.0 ) {
  for ( d = 0; d < native_data.length; d++ ) {
    native_data[d] = native_data[d] * slope + inter; 
  }
} 

for ( d = 0; d < native_data.length; d++ ) {
  value = native_data[d];
  if ( value > n_max ) {
    n_max = value;
  }
  if ( value < n_min ) {
    n_min = value;
  }
}
```
If I'm right it does the same thing and I think it's more readable. What do you think ?